### ISSUE TITLE
Expose Cucumber version

### ISSUE THREAD

**Issue by jan-molak** [cucumber/cucumber-js#1853]
**Is your feature request related to a problem? Please describe.**

[Serenity/JS](https://serenity-js.org) currently supports Cucumber versions 1.x to 7.x and I'd like it to be able to support version 8.x as well.

In order to do that, Serenity/JS needs to know what version of Cucumber it's being used with.

At the moment, Serenity/JS figures out the version based on the `version` field present in Cucumber's `package.json` file.

However, in Cucumber 8 RC1 `package.json` is not listed in `exports`, so trying to `require('@cucumber/cucumber/package.json')` results in:

```
 Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './package.json' is not defined by "exports"
```

**Describe the solution you'd like**

It would be great if Cucumber 8 could either:
- added `package.json` to its exports so that Serenity/JS could read the version from it
- exported a `version` const so that it could be imported by consumer frameworks (e.g. `import { version } from '@cucumber/cucumber'`)

**Describe alternatives you've considered**

An alternative solution would be for Serenity/JS to check if `@cucumber/cucumber` module is present, and if it is and it's not possible to load its `package.json` => assume it's version 8.

It _could_ work, but it smells of a an awful hack ;-)

**Additional context**
N/A

**Comment by aurelien-reeves** [comment]
Indeed, we should try to find a way before releasing final version of 8.0.0

Thanks for the report!

**Comment by jan-molak** [comment]
Thanks, give me a shout if you'd like me to try it again with any new RCs.

**Comment by jan-molak** [comment]
Nice one, thanks @aurelien-reeves, #1866 looks great.
I'm invoking Cucumber.js [programmatically](https://github.com/serenity-js/serenity-js/blob/0c706f753343f1ce35b7e77460062d5762f6d4aa/packages/cucumber/src/cli/CucumberCLIAdapter.ts#L101-L117) rather than spawning a new process, so importing a `version` is preferable.
